README.md

```diff
@@ -38,3 +38,7 @@ moduleGraphAssert {
 ```
 ./gradlew generateModulesGrapvizText -Pmodules.graph.print.statistics=true -Pmodules.graph.of.module=:feature-one
 ```
+- Adding parameter `modules.graph.output.gv` saves the graphViz file to specified path (extension `.gv` is added if necessary)
```

Comment:
```suggestion
- Adding parameter `modules.graph.output.gv` saves the graphViz file to specified path
```

Reply:
please apologize me, I forgot to edit the README.md

Reply:
no worries, it is still there though :) I think to accept suggestion you need to commit it, not resolve the conversation :D